src/components/share-button/share-button-component.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ReactComponent as ShareIcon } from 'icons/share.svg';
+import cx from 'classnames';
+import ReactTooltip from 'react-tooltip';
+
+import styles from './share-button-styles.module';
+
+const ShareButtonComponent = (props) => {
+  const { theme, variant = 'icon', setShareModalOpen } = props;
```

Comment:
Would it make sense to allow the component to consume a `text` `prop` to display by the button instead of having fixed `variants`?

Reply:
I thought about it. But the text, in this case, is still always linked to the variant. We need these three different variants as for example the icon is in different positions. Maybe we could use this text prop but the markdown is already different in each variant

Reply:
:ok_hand:  fair enough